src/fee_stats/mod.rs

```diff
@@ -0,0 +1,33 @@
+pub mod fee_stats_request;
+pub mod response;
+
+static FEE_STATS_PATH: &str = "fee_stats";
+
+pub mod prelude {
+    pub use super::fee_stats_request::*;
+    pub use super::response::*;
+}
+
+#[cfg(test)]
+mod tests {
+    use super::prelude::*;
+    use crate::horizon_client::HorizonClient;
+
+    #[test]
+    fn dummy_test() {
+        assert_eq!(super::FEE_STATS_PATH, "fee_stats");
+    }
```

Comment:
Deze kan weg denk ik. Het pad zal ongetwijfeld geraakt worden in 1 van de andere tests als het goed is.